scipy-notebook/Dockerfile

```
@@ -50,9 +50,9 @@ RUN conda install --quiet --yes \
     # Also activate ipywidgets extension for JupyterLab
     # Check this URL for most recent compatibilities
     # https://github.com/jupyter-widgets/ipywidgets/tree/master/packages/jupyterlab-manager
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager@^1.1.0 --no-build && \
-    jupyter labextension install jupyterlab_bokeh@1.0.0 --no-build && \
-    jupyter labextension install jupyter-matplotlib --no-build && \
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager@^2.0.0 --no-build && \
+    jupyter labextension install @bokeh/jupyter_bokeh@^2.0.0 --no-build && \
+    jupyter labextension install jupyter-matplotlib@0.7.2 --no-build && \
```

I'm fine with either pinning (`bokeh@2.0.0` and `jupyter-matplotlib@0.7.2` exactly) or (`bokeh@^2.0.0` and `^jupyter-matplotlib@0.7.2`). Consistency in how we're pinning across packages makes tends to make problems easier to debug when they arise.

```suggestion
    jupyter labextension install jupyter-matplotlib@^0.7.2 --no-build && \
```

---

@mpmdean I'm going to assume you're OK with the above change and just want the problem fixed, so I'm hitting the "commit suggestion" button on your behalf to get the ball rolling. 😄